torch/tensor.py

```diff
@@ -904,6 +904,45 @@ def rename(self, *names, **rename_map):
         # See Note [rename_ / rename API]
         return update_names(self, names, rename_map, inplace=False)
 
+    def to_sparse_csr(self):
+        """ Convert a dense, strided tensor to compressed row storage format. 
+        Only works with 2D tensors.
+
+        Examples::
+
+            >>> dense = torch.randn(5, 5)
+            >>> sparse = dense.to_sparse_csr()
+            >>> sparse._nnz()
+            3125
+
+        """
+        shape = self.size()
+        fill_value = 0
+        if len(shape) != 2:
+            raise RuntimeError("Only 2D tensors can be converted to the CSR format but got shape: ", shape)
+
+        if self.is_sparse:
+            return self.to_dense().to_sparse_csr()  # TODO: optimize this.
```

Comment:
I think direct coo->csr conversion should be a high priority task.

Reply:
Done.